samples/bluetooth/bap_broadcast_source/sample.yaml

```diff
@@ -14,6 +14,17 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
     tags: bluetooth
     sysbuild: true
+  sample.bluetooth.bap_broadcast_source.nrf5340_audio_dk_cpuapp.fem:
+    harness: bluetooth
+    platform_allow:
+      - nrf5340_audio_dk/nrf5340/cpuapp
+    integration_platforms:
+      - nrf5340_audio_dk/nrf5340/cpuapp
+    extra_args:
+      - DTC_OVERLAY_FILE=boards/nrf5340_audio_dk_nrf5340_cpuapp_nrf21540_ek.overlay
+      - hci_ipc_DTC_OVERLAY_FILE=boards/nrf5340_audio_dk_nrf5340_cpunet_nrf21540_ek.overlay
```

Comment:
One thing I've experimented with in https://github.com/zephyrproject-rtos/zephyr/pull/87456 is to automatically apply additional specific files based on a sysbuild Kconfig option in the sysbuild.cmake files. 

That would make it so that they are applied automatically with just `west build --sysbuild`. 

Not saying that it's an approach that should be used here, but worth considering something like that which could perhaps be consider here, like adding a `SB_CONFIG_FEM` or something

Reply:
Yes, though about it and can be good for now.

As of today, all I know is to copy-paste `sysbuild.cmake` and `Kconfig.sysbuild` into `samples/bluetooth/<sample>` folders; this is not scalable for many reasons. Need to switch to a common solution, hence will keep this PR to a simple change to cover the overlays in CI using the already existing sysbuild files in this sample.

Reply:
Yeah. It's a common issue that the current way of building easily becomes complex, and hard to tell how to do it "properly". 

We can "fix" that by extending the sysbuild.cmake (or other .cmake files) where we can configure more on what is being added and when. 

Alternatively, the sample.yaml files also do a good job of describing it, but they are not applied/used when using `west build`. It would be nice if we could provide some file that handles applying the different required/recommended config files when building, as the `prj.conf` approach is simply not expressive enough